app/assets/stylesheets/helpers/_header.scss

```diff
@@ -472,76 +472,104 @@
   display: none;
 }
 
-$light-blue: #259EDA;
+$covid-green: #7eff8e;
 
 .global-bar {
-  background-color: #E5EAF1;
-  border-top: 8px solid $govuk-brand-colour;
+  background-color: govuk-colour("black");
+  border-top: 5px solid $covid-green;
   display: none;
-  padding: govuk-spacing(3) 0;
+  padding: govuk-spacing(5) 0;
 
   .show-global-bar & {
     display: block;
   }
 
+  .govuk-link,
+  .govuk-link:link {
+    color: govuk-colour("white");
+
+    &:visited {
+      color: govuk-colour("white");
+    }
+
+    &:focus {
+      color: govuk-colour("black");
+    }
+  }
+
   .global-bar-message-container {
     @include core-19;
     @extend %site-width-container;
     position: relative;
+    color: govuk-colour("white");
 
-    .global-bar__dismiss {
-      display: inline-block;
-      position: relative;
-      margin-top: govuk-spacing(4);
+    .global-bar-message {
+      margin-bottom: 0;
+      margin-top: 0;
+
+      .global-bar-title {
+        font-weight: 700;
+        margin-right: govuk-spacing(2);
+        margin-bottom: govuk-spacing(1);
+
+        &:only-child {
+          margin: 0;
+        }
+      }
     }
 
-    @include govuk-media-query($from: tablet) {
-      .global-bar__dismiss {
-        position: absolute;
+    .global-bar__dismiss {
+      display: none;
+
+      &--show {
```

Comment:
non-blocker, we tend to explicitly expand these `.global-bar__dismiss--show` to be able to find them quickly using search.